pkg/agent/core/ngt/handler/grpc/index_test.go

```diff
@@ -182,6 +182,22 @@ func Test_server_SaveIndex(t *testing.T) {
 		}
 		return nil
 	}
+
+	/*
+		- Equivalence Class Testing (with copy on write disable)
+			- case 1.1: success to save 1 inserted index
+			- case 1.2: success to save 100 inserted index
+			- case 2.1: fail to save index with no write access on backup folder
```

Comment:
please add the case
- fail to save index when other save index process is running

Reply:
```suggestion
		- Equivalence Class Testing (with copy on write disable)
			- case 1.1: success to save 1 inserted index
			- case 1.2: success to save 100 inserted index
			- case 2.1: fail to save index with no write access on backup folder
			- case 3.1: fail to save index when other save index process is running
```